homework_4/index.js

```diff
@@ -0,0 +1,45 @@
+const express = require('express');
+const app = express();
+const logger = require('morgan');
+const cohorts_route = require('./routes/cohorts');
+
+const PORT = 3000;
+const LOCALHOST = 'localhost';
+
+/* Logging software */
+app.use(logger('dev'));
+
+app.use(express.urlencoded({ extended: true }));
+
+const path = require('path');
+app.use(express.static(path.join(__dirname, 'public')));
+
+const methodOverride = require('method-override');
+
+app.use(
+  methodOverride((req, res) => {
+    if (req.body && req.body._method) {
+      const method = req.body._method;
+      return method;
+    }
+  })
+);
+
+/* View Engine */
+app.set('view engine', 'ejs');
+app.set('views', 'views');
+
+app.get('/', (req, res) => {
+  res.send('Hello from the the other side 👾');
```

Comment:
Root route is pretty much the landing page in this app and users should be able to get an access to nav bar in order to be able to explore the app. If you didn't want to design the home page you could redirect to the index page.